tools/make/bakeddata/src/main.rs

```diff
@@ -131,30 +131,6 @@ fn main() {
             .unwrap();
         }
 
-        // Crates with non-singleton markers need fallback
```

Comment:
Question: Should `icu_locale` be added (perhaps added back) to the ctx crates that get printed? Or is that not relevant here at all

Reply:
I don't think we print anymore? Either way, it's more complicated than being a ctx crate now because of the `icu` module wrapping

Reply:
Aha, this already gets handled in the baked exporter.

This code is just for the ICU4X repo which does some fancier things.